build/distributions/CentOS/6/x86_64/DVD/ks/pupclient_x86_64.cfg

```diff
@@ -14,8 +14,16 @@
 #  > ruby -r 'digest/sha2' -e 'puts "password".crypt("$6$" + rand(36**8).to_s(36))'
 #
 # Optionally:
-# - Adjust the bootloader line for UEFI boot per inline instructions below
-# - Disable FIPS per inline instructions below
+#   - Enable transparent disk encryption by passing 'simp_disk_crypt' in the
```

Comment:
What does 'transparent disk encryption' mean?  Transparent seems to be the opposite of encrypted.

Reply:
Transparent as in the user doesn't notice that it is there, and the files are encrypted, not just the store.

Reply:
Yup, transparent as in "meets the technical requirement if you need it but it isn't great"